relay-event-schema/src/protocol/contexts/trace.rs

```diff
@@ -118,6 +118,10 @@ pub struct TraceContext {
     /// This flag only applies to events with [`Error`] type that have an associated dynamic sampling context.
     pub sampled: Annotated<bool>,
 
+    /// Arbitrary additional data on a trace.
+    #[metastructure(pii = "true")]
```

Comment:
```suggestion
    #[metastructure(pii = "true", bag_size = "medium")]
    #[metastructure(skip_serialization = "empty")]
```

I don't know if this is the right bag size, but I noticed other data bags have it + the `skip_serialization` attr.

Reply:
@jjbayer  changed the annotations in [e2a23c3](https://github.com/getsentry/relay/pull/2899/commits/e2a23c3dc2a355701135cdd6fd749a10641cb76d)